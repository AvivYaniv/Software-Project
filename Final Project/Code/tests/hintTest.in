edit tests/hint_board.txt
hint 1 1
solve tests/hint_board.txt
set 1 1 1
hint 2 10
hint 10 2
hint 2 2
hint 1 1
hint 1
hint 1 a
hint a 2
set 4 1 1 
print_board
hint 2 1
undo
autofill
set 8 9 4
validate
hint 2 1
undo
hint 2a 1
hint 2 a1
solve tests/b2_2_almostComp.txt
autofill
hint 1 3
exit