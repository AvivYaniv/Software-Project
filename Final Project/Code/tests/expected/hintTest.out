Sudoku
------
Enter your command:
----------------------------------------
|            |      6   7 |      8     |
|      8     |          3 |            |
|            |  4   8     |          5 |
----------------------------------------
|            |            |          4 |
|  9   5     |          4 |          3 |
|            |      9     |  5       2 |
----------------------------------------
|      9   3 |      4     |  7         |
|  8   2   5 |  7   3   1 |  9         |
|            |  9   5     |  3         |
----------------------------------------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|            |      6.  7.|      8.    |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|      9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.        |
|            |  9.  5.    |  3.        |
----------------------------------------
Enter your command:
----------------------------------------
|  1         |      6.  7.|      8.    |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|      9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.        |
|            |  9.  5.    |  3.        |
----------------------------------------
Enter your command:
Error: value not in range 1-9
Enter your command:
Error: value not in range 1-9
Enter your command:
Error: cell is fixed
Enter your command:
Error: cell already contains a value
Enter your command:
ERROR: invalid command
Enter your command:
Error: value not in range 1-9
Enter your command:
Error: value not in range 1-9
Enter your command:
----------------------------------------
|  1*        |  1*  6.  7.|      8.    |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|      9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.        |
|            |  9.  5.    |  3.        |
----------------------------------------
Enter your command:
----------------------------------------
|  1*        |  1*  6.  7.|      8.    |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|      9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.        |
|            |  9.  5.    |  3.        |
----------------------------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
----------------------------------------
|  1         |      6.  7.|      8.    |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|      9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.        |
|            |  9.  5.    |  3.        |
----------------------------------------
Undo 4,1: from 1 to _
Enter your command:
Cell <9,1> set to 9
Cell <1,7> set to 6
Cell <9,8> set to 6
----------------------------------------
|  1         |      6.  7.|      8.  9 |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|  6   9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.      6 |
|            |  9.  5.    |  3.        |
----------------------------------------
Enter your command:
----------------------------------------
|  1         |      6.  7.|      8.  9 |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|  6   9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.      6 |
|            |  9.  5.    |  3.  4     |
----------------------------------------
Enter your command:
Validation failed: board is unsolvable
Enter your command:
Error: board is unsolvable
Enter your command:
----------------------------------------
|  1         |      6.  7.|      8.  9 |
|      8.    |          3.|            |
|            |  4.  8.    |          5.|
----------------------------------------
|            |            |          4.|
|  9.  5.    |          4.|          3.|
|            |      9.    |  5.      2.|
----------------------------------------
|  6   9.  3.|      4.    |  7.        |
|  8.  2.  5.|  7.  3.  1.|  9.      6 |
|            |  9.  5.    |  3.        |
----------------------------------------
Undo 8,9: from 4 to _
Enter your command:
Error: value not in range 1-9
Enter your command:
Error: value not in range 1-9
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|        |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Cell <1,2> set to 4
Cell <2,3> set to 4
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|      4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Hint: set cell to 3
Enter your command:
Exiting...
